src/cljs/athens/electron/dialogs.cljs

```diff
@@ -0,0 +1,96 @@
+(ns athens.electron.dialogs
+  (:require
+    [athens.athens-datoms :as athens-datoms]
+    [athens.db :as db]
+    [athens.electron.utils :as utils]
+    [datascript.core :as d]
+    [datascript.transit :as dt]
+    [re-frame.core :as rf]))
+
+
+(def electron (js/require "electron"))
+(def remote (.. electron -remote))
+(def fs (js/require "fs"))
+(def dialog (.. remote -dialog))
+(def path (js/require "path"))
+
+
+
+(rf/reg-event-fx
+  :fs/open-dialog
+  (fn [{:keys [db]} _]
+    (js/alert (str "No DB found at " (:db/filepath db) "."
+                   "\nPlease open or create a new db."))
+    {:dispatch-n [[:modal/toggle]]}))
+
+
+(defn move-dialog!
+  "If new-dir/athens already exists, no-op and alert user.
+  Else copy db to new db location. When there is an images folder, copy /images folder and all images.
+    file:// image urls in block/string don't get updated, so if original images are deleted, links will be broken."
+  []
+  (let [res     (.showOpenDialogSync dialog (clj->js {:properties ["openDirectory"]}))
+        new-dir (first res)]
+    (when new-dir
+      (let [curr-db-filepath @(rf/subscribe [:db/filepath])
+            base-dir         (.dirname path curr-db-filepath)
+            base-dir-name    (.basename path base-dir)
+            curr-dir-images  (.resolve path base-dir utils/IMAGES-DIR-NAME)
+            new-dir          (.resolve path new-dir base-dir-name)
+            new-dir-images   (.resolve path new-dir utils/IMAGES-DIR-NAME)
+            new-db-filepath  (.resolve path new-dir utils/DB-INDEX)]
+        (if (.existsSync fs new-dir)
+          (js/alert (str "Directory " new-dir " already exists, sorry."))
+          (do (.mkdirSync fs new-dir)
+              (.copyFileSync fs curr-db-filepath new-db-filepath)
+              (rf/dispatch [:db-picker/move-db curr-db-filepath new-db-filepath])
+              (rf/dispatch [:db/update-filepath new-db-filepath])
```

Comment:
Not sure these effects are right. Can definitely remove `:db/update-filepath`, which is deprecated in this PR.